### PR TITLE
Add POT config for ocr-lite-hrnet-18

### PR DESCRIPTION
**Pull request by evgeniya-egupova**
This PR adds compression config for more accurate post-training quantization of ocr-lite-hrnet-18

### CODE REVIEW

File: configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                        "Add_3570",
+                        "Add_3598",
+                        "Add_3619",
+                        "Add_3656"
+                    ]
+                }
+            }
+        }
+    ]
+}
```
Comment (AlexanderDokuchaev) [inline configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json]: newline

File: configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json [hunk 2]
```diff
@@ -0,0 +1,92 @@
+{
+    "algorithms": [
+        {
```
Comment (AlexanderDokuchaev) [inline configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json:3]: Is cant used json like POT config format without "algorithms"? Or can be another parameters here?

**Review by alexsu52 — APPROVED** [review]
It looks good to me

File: configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json (hunk 2 above)
Comment (evgeniya-egupova) [reply configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json:3]: Algorithms in POT are always in a list. Can't start json config with a list.

Comment (AlexanderDokuchaev) [reply configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json:3]: ok

**Review by AlexanderDokuchaev — APPROVED** [review]

**Review by evgeny-izutov — APPROVED** [review]

**Pull request merged**